amun/api_v1.py

```diff
@@ -108,17 +120,24 @@ def _adjust_default_requests(dict_: dict) -> None:
     dict_['requests'] = new_requests
 
 
+@versionchanged(
```

Comment:
this is a connexion decoration?

Reply:
It's not. I could not find any connexion specific decorator for that. The `deprecated` package seems to be used commonly.